client/lib/plans/constants.js

```diff
@@ -464,7 +459,16 @@ export const PLANS_LIST = {
 	[ PLAN_BUSINESS ]: {
 		...getPlanBusinessDetails(),
 		term: TERM_ANNUALLY,
-		getBillingTimeFrame: WPComGetBillingTimeframe,
+		getBillingTimeFrame: abtest => {
+			if ( isEnabled( 'upgrades/2-year-plans' ) ) {
+				if ( abtest && abtest( 'multiyearSubscriptionsBusinessPriceTest' ) === 'show' ) {
+					return i18n.translate( '/month, billed annually or every two years' );
+				}
+				return i18n.translate( '/month, billed annually' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( '/month, billed annually or biennially' )` *ES Score: 17*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/2-year-subscriptions-test-only-business-plans)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).